NEWS.md

```diff
@@ -1,3 +1,22 @@
+# RcppEnsmallen 0.2.16.2.1
+
+- Upgraded to ensmallen 2.16.2: "Severely Dented Can Of Polyurethane" (2021-03-25)
+  - Fix CNE test trials
+    ([#267](https://github.com/mlpack/ensmallen/pull/267)).
+  - Update Catch2 to 2.13.4
+    ([#268](https://github.com/mlpack/ensmallen/pull/268)).
+  - Fix typos in documentation
+    ([#270](https://github.com/mlpack/ensmallen/pull/270),
+     [#271](https://github.com/mlpack/ensmallen/pull/271)).
+  - Add clarifying comments in problems/ implementations
+    ([#276](https://github.com/mlpack/ensmallen/pull/276)).
+  - Remove `AdamSchafferFunctionN2Test` test from Adam test suite to prevent
+    spurious issue on some aarch64 ([#265](https://github.com/mlpack/ensmallen/pull/259)).
+  - Fix implementation of fonesca fleming problem function f1 and f2
+  - Fix implementation of fonesca fleming problem function f1 and f2 
+ - Add optimizer summary report callback
+ - Add optimizer summary report callback 
+
```

Comment:
```suggestion
# RcppEnsmallen 0.2.16.2.1 (GitHub-only Release)

- Upgraded to ensmallen 2.16.2: "Severely Dented Can Of Polyurethane" (2021-03-25)
  - Fix CNE test trials
    ([#267](https://github.com/mlpack/ensmallen/pull/267)).
  - Update Catch2 to 2.13.4
    ([#268](https://github.com/mlpack/ensmallen/pull/268)).
  - Fix typos in documentation
    ([#270](https://github.com/mlpack/ensmallen/pull/270),
     [#271](https://github.com/mlpack/ensmallen/pull/271)).
  - Add clarifying comments in problems/ implementations
    ([#276](https://github.com/mlpack/ensmallen/pull/276)).
  - Remove `AdamSchafferFunctionN2Test` test from Adam test suite to prevent
    spurious issue on some aarch64 ([#265](https://github.com/mlpack/ensmallen/pull/259)).
  - Fix implementation of fonesca fleming problem function f1 and f2
  - Add optimizer summary report callback 
```